src/rules/flake8_bandit/settings.rs

```diff
@@ -34,11 +34,19 @@ pub struct Options {
     /// A list of directories to consider temporary, in addition to those
     /// specified by `hardcoded-tmp-directory`.
     pub hardcoded_tmp_directory_extend: Option<Vec<String>>,
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = "check-typed-exception = true"
+    )]
+    /// A list of directories to consider temporary.
```

Comment:
This looks like it might be incorrect for this option. Maybe copied over from elsewhere?